skl2onnx/algebra/automation.py

```diff
@@ -1,8 +1,5 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-# --------------------------------------------------------------------------
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@szha I noticed the copyrights in many files were removed as part of the license change. Was it on purpose or unintentionial?

Reply:
This is intentional. We can include the copyright notices into a unified NOTICE file for simplicity.